spec.html

```diff
@@ -32725,7 +32697,7 @@ <h2>Syntax</h2>
 
           TemporalSign :::
             ASCIISign
-            &lt;MINUS&gt;
+            &lt;U+2212 MINUS SIGN&gt;
```

Comment:
This is fine, but we typically parenthesise the code point name.

```suggestion
            &lt;U+2212 (MINUS SIGN)&gt;
```

Reply:
We're not consistent about the parentheses, although it's true that they are much more common than not (exceptions being concentrated in [Canonicalize](https://tc39.es/ecma262/multipage/text-processing.html#sec-runtime-semantics-canonicalize-ch), plus one outlier in [Appendix F](https://tc39.es/ecma262/multipage/additions-and-changes-that-introduce-incompatibilities-with-prior-editions.html#sec-additions-and-changes-that-introduce-incompatibilities-with-prior-editions)).

But this particular suggestion also has the drawback that it will be rejected by the just-released version of ESMeta (which AFAICT was only [extended](https://github.com/es-meta/esmeta/pull/148/files#diff-89c5a6ddf42547b2b445dd5b83999148daa58482bfdbaea610ff428e37629f1eR132) to allow `<U[+]([1-9A-F]|10)?[0-9A-F]{4}( \w+)*>`), and beyond that I'd like to get to a point where ECMA-262 and 402 hew more closely to [Unicode conventions](https://www.unicode.org/versions/Unicode15.0.0/appA.pdf) (i.e., "U+…" followed by an unpunctuated uppercase name [in small caps where possible]). Given that, is appearance inside a grammar as a nonterminal sufficient distinction for omitting the parentheses here?

Reply:
I think either way is fine, I was just trying to match our current conventions. I'll bring it up in editor call.